server/Orders.js

```diff
@@ -0,0 +1,134 @@
+/* Orders is observer of: Inventory
+ *           observed by: Drivers
+ */
+class Orders {
+    constructor() {
+        this.orders = [];
+        this.drivers = [];
+        this.inventories = [];
+    }
+
+    fillList(ordersList) {
+        ordersList.forEach(order => {
+            order = new Order(order);
+            if (!this.orders.includes(order)) {
+                this.orders.push(order);
+            }
+        })
+    }
+
+    // Beginning of Observable's duties
+    notify(URL) {
```

Comment:
this notification to the food bank is only an order status update, so we dont need send a lot of this pay load. The order iD and that order's progress status is enough. They already know the inventory they requested, their own foodBank and they can get the groceryId from their foodBankId